apps/docs/docs/learn/03-installation.mdx

```diff
@@ -196,6 +196,48 @@ provides plugins for Webpack, Rollup, and Next.js.
 
 </details>
 
+<details>
```

Comment:
Added _esbuild_ plugin example to the installations page in docs.

Reply:
Please be patient as it may take me a few days to review this properly.